src/app/components/ATIAnalytics/params/index.ts

```diff
@@ -77,6 +79,7 @@ const pageTypeUrlBuilders = {
   [UGC_PAGE]: noOp,
   [AV_EMBEDS]: noOp,
   [DOWNLOADS_PAGE]: noOp,
+  [AUDIO_PAGE]: buildTvRadioATIParams,
```

Comment:
This should be `noOp` instead of `buildTvRadioATIParams` - I am sure it won't really make a difference, but it will make the clean up of this analytics code so much easier once all pages are fetching their data from the BFF

```suggestion
  [AUDIO_PAGE]: noOp,
```